Serve.php

```diff
@@ -0,0 +1,24 @@
+
+class Serve
+{
+  private $xml_data = null;
+
+  function __construct($format)
+  {
+    header("Access-Control-Allow-Origin: *");
+    header("Content-Type: application/json; charset=UTF-8");
+    header("Access-Control-Allow-Methods: GET");
+    header("Access-Control-Allow-Headers: *");
+
+    if ($format === 'json') {
```

Comment:
constructor is used only for preparing code for the whole class but, please don't write login in it,
methods can handle it.